data/meson.build

```diff
@@ -30,25 +30,47 @@ if get_option('daemon')
   )
 endif
 
-con2 = configuration_data()
-con2.set('libexecdir', libexecdir)
-con2.set('bindir', bindir)
-con2.set('localstatedir', localstatedir)
-con2.set('datadir', datadir)
-con2.set('sysconfdir', default_sysconfdir)
+if get_option('systemd')
+  con2 = configuration_data()
+  con2.set('libexecdir', libexecdir)
+  con2.set('bindir', bindir)
+  con2.set('datadir', datadir)
 
-# replace @libexecdir@
-configure_file(
-  input : 'org.freedesktop.fwupd.service.in',
-  output : 'org.freedesktop.fwupd.service',
-  configuration : con2,
-  install: true,
-  install_dir: join_paths(datadir,
-                          'dbus-1',
-                          'system-services'),
-)
+  rw_directories = [join_paths (localstatedir, 'lib', 'fwupd'),
```

Comment:
I think:

    rw_directories = []
    rw_directories += join_paths (localstatedir, 'lib', 'fwupd')
    rw_directories += join_paths (default_sysconfdir, 'fwupd', 'remotes.d')

makes this a lot cleaner with a lot less indenting.

Reply:
Fixed